zigzag/xml_parsing_facade.py

```diff
@@ -65,7 +65,7 @@ def _determine_ci_environment(self):
         except AttributeError:
             self._mediator.ci_environment = 'asc'  # hard code this here
 
-    def _read(self):
+    def _read(self, file_path):
```

Comment:
Can you update docstring to include file path or change it back so that it gets the file path from the mediator.

Reply:
I'll ticket this (lazy load cache pattern)